components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1818,7 +1806,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
         const user = await this.checkAndBlockUser("openPort");
 
-        const workspace = await this.workspaceService.getWorkspace(user.id, workspaceId);
+        const { workspace } = await this.workspaceService.getWorkspace(user.id, workspaceId);
```

Comment:
use the running instance

Reply:
findRunningInstance vs getCurrentInstance. I'd rather not remodel that behavior we already have in svc.openPort, and just drop the old code after the rollout.